modules/apps/object/object-web/src/main/resources/META-INF/resources/object_entries/object_entry/form.jsp

```diff
@@ -246,10 +246,18 @@ portletDisplay.setURLBack(backURL);
 							scheduleContainerInput.value
 						);
 
+						if (isPastDate(scheduleContainerInputValue.displayDate)) {
+							shouldSubmitForm = false;
```

Comment:
Hi @larissacribeiro, in some situation we can get here using a past `displayDate`? 

```
// handleError:
...
		const date = new Date(value);

		if (date <= new Date()) {
			setDateError(
				Liferay.Language.get('the-date-entered-is-in-the-past')
			);

			return false;
		}
```
		
```
const handleSubmit = (
		event: React.MouseEvent<HTMLButtonElement, MouseEvent>
	) => {
		const isValid = handleError(displayDate);

		if (!isValid) {
			return;
		}
...

        callWindowGlobalFunction(submitRef);
```

As I understand, we will never call the callWindowGlobalFunction() function if handleError returns any error.